src/Microsoft.AspNetCore.Html.Abstractions/HtmlString.cs

```diff
@@ -10,18 +10,23 @@ namespace Microsoft.AspNetCore.Html
     /// <summary>
     /// An <see cref="IHtmlContent"/> implementation that wraps an HTML encoded <see cref="string"/>.
     /// </summary>
-    public class HtmlEncodedString : IHtmlContent
+    public class HtmlString : IHtmlContent
     {
         /// <summary>
-        /// An <see cref="IHtmlContent"/> instance for <see cref="Environment.NewLine"/>.
+        /// An <see cref="HtmlString"/> instance for <see cref="Environment.NewLine"/>.
         /// </summary>
-        public static readonly IHtmlContent NewLine = new HtmlEncodedString(Environment.NewLine);
+        public static readonly HtmlString NewLine = new HtmlString(Environment.NewLine);
```

Comment:
@rynowak and I talked about this and it was an oddity/inconsistency in the API. An `HtmlString.NewLine`/`HtmlString.Empty` returning something other than a type representing itself is odd. It'd kind of be like `string.Empty` returning `object` instead of `string`. 

Conclusion: since we added the `Empty` type from Mvc (a direct copy) it brought to light the weirdness of `NewLine` returning type `IHtmlContent`.